components/Buttons/tests/unit/ButtonsTests.m

```diff
@@ -1116,6 +1116,19 @@ - (void)testAdjustsFontProperty {
                              @"Font size should be equal to MDCFontTextStyleButton's.");
 }
 
+- (void)testAdjustFontForContentSizeCategoryWhenScaledFontIsUnavailableDefaultBehavior {
+  // Then
+  XCTAssertTrue(self.button.adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable);
+}
+
+- (void)testAdjustsFontForContentSizeCategoryWhenScaledFontIsUnavailableSetsValueCorrectly {
+  // When
+  self.button.adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable = NO;
+
+  // Then
+  XCTAssertFalse(self.button.adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable);
+}
+
```

Comment:
Can we add a test that tests the expected code path for adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable? E.g. two separate tests, one for the false behavior, one for the true behavior, that validates that the fallback behavior works as expected?

Reply:
Snapshot test have been added, these changes are all reverted. The snapshot test cover when `adjustFont....` is both `YES` and `NO`.